src/instance/SectionInstance.ts

```diff
@@ -71,10 +73,22 @@ export default class SectionInstance implements Child, Pluggable {
         const searchPath = field instanceof ChoiceField ? _.initial(path) : path;
         let instanceData = null;
 
+        const noMatchHandler = (stopInstance: Instance, remaining: PathSegment[]) => {
```

Comment:
`remaining` should just be a `Path` which you can import from `yinz` without importing from dist

Reply:
Ah yep, I let VS Code import it and that's what I get for not checking it